.github/workflows/track-contributor-prs.yml

```
@@ -10,27 +10,14 @@ permissions:
-  
   update-pr-count:
-    needs: calculate-changes
     if: github.event.pull_request.merged == true
     uses: StabilityNexus/ContributorAutomation/.github/workflows/reusable-update-pr.yml@main
     with:
       pr_number: ${{ github.event.pull_request.number }}
       repo_name: ${{ github.repository }}
       pr_author: ${{ github.event.pull_request.user.login }}
       pr_title: ${{ github.event.pull_request.title }}
-      lines_changed: ${{ needs.calculate-changes.outputs.lines_changed }}
+      lines_changed: ${{ github.event.pull_request.additions + github.event.pull_request.deletions }}
```

actionlint (1.7.11)

[error] got unexpected character '+' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)